### ISSUE TITLE
how to use Kolors-IP-Adapter-FaceID-Plus

### ISSUE THREAD

**Issue by sipie800** [cubiq/ComfyUI_IPAdapter_plus#666]
As the commit mentions, the faceid of kolors is supported. But I can't get the weight loaded. Which loader should be used ? Use the ipadapter_kolors.json example and it can‘t use weight from hf Kwai-Kolors/Kolors-IP-Adapter-FaceID-Plus.

_Exception: invalid IPAdapter model D:\yc\ComfyUI\models\ipadapter\kolors_ipa-faceid-plus.bin_

please provide a simple workflow if possible.

**Comment by sipie800** [comment]
Sorry. I mistakes kolors IPA to FaceId. It's not supported yet.

**Comment by cubiq** [comment]
faceid was just released, it should be easy to port

**Comment by Remember2015** [comment]
I guess FaceID-Plus of Kolors used clip-336, which is different from original FaceID-Plus?

**Comment by WingeD123** [comment]
> I guess FaceID-Plus of Kolors used clip-336, which is different from original FaceID-Plus?

yes, different from other faceid, but same as kolors' ipadapter

**Comment by cubiq** [comment]
faceid plus uses the embeds from both the clip vision (at 336) and insightface. That unfortunately makes the model for non-commercial use only.

I tried quickly to port it today, the model works but the results are not very good, I have to check if I need to do something else for proper support

**Comment by wailovet** [comment]
> faceid plus uses the embeds from both the clip vision (at 336 in case of Kolors) and insightface. That unfortunately makes the model for non-commercial use only.
> 
> I tried quickly to port it today, the model works but the results are not very good, I have to check if I need to do something else for proper support

I also attempted to transplant one version, and it was equally not very good.

**Comment by sipie800** [comment]
it's possible if it's not good as the official claims

**Comment by sipie800** [comment]
it's not good. I tried and figured out it does nothing more than copypasting the given face. Maybe the node has done something wrong?

**Comment by cubiq** [comment]
yes I believe the implementation is not entirely correct, I'll do some more testing in the coming days
